roles/setup_sushy_tools/tasks/main.yml

```diff
@@ -68,7 +68,7 @@
             cert_file_prefix: sushy_tools
             cert_path: "{{ sushy_cert_dir }}/sushy_tools_{{ inventory_hostname }}.crt"
             privatekey_path: "{{ sushy_cert_dir }}/sushy_tools.key"
-            cert_common_name: "{{ ansible_fqdn }}"
+            cert_common_name: "{{ sushy_fqdn }}"
```

Comment:
This file and roles/validate_inventory/tasks/vms.yml look like they were part of PR46. Did you intend for them to be here?

Reply:
It probably should have been PR46. i noticed it didn't exist while I was documenting stuff so I just added it in here as its only 2 lines it - it has is own commit though.